packages/react-toast/src/components/Body/styled.ts

```diff
@@ -9,8 +9,8 @@ export const iconStyles = ({ theme }) =>
       objectFit: "contain",
       alignSelf: "center",
       padding: "2px",
-      maxHeight: "35px !important",
-      maxWidth: "35px !important",
+      maxHeight: "40px !important",
```

Comment:
is !important really necessary?

Reply:
yeah it is,  i removed it at one point at it screwed something up

Reply:
i think we can fix it but for now i think its ok